src/screens/ExchangeModal.tsx

```diff
@@ -943,6 +1025,30 @@ export default function ExchangeModal({
     ? !!inputCurrency
     : !!inputCurrency && !!outputCurrency;
 
+  const handleConfirmExchangePress = useCallback(() => {
+    if (!outputNetworkDetails?.balance?.amount) return NOOP;
```

Comment:
for when native asset is not loaded yet?

Reply:
Yeah it mostly to satisfy Typescript, even though we know it wont ever be undefined because this function wont be called unless its available